src/components/pages/Stake/validation.ts

```diff
@@ -12,7 +13,9 @@ const validateLock = ({
   ) {
     return [false, Reasons.FetchingData];
   }
-
+  if (transactionType === TransactionType.IncreaseLockTime) {
+    return [true];
```

Comment:
Is this always the case? What if I set a null value? What if it's under 6 days? We're also using the UI to guard against the case where I don't have a staking balance yet, so we can guard against that case here, too.